### ISSUE TITLE
Version 1.11.3.3 released without updating the code

### ISSUE THREAD

**Issue by nikolay** [pagespeed/ngx_pagespeed#1246]
https://github.com/pagespeed/ngx_pagespeed/releases/tag/v1.11.33.3-beta

**Comment by oschaaf** [comment]
I have not seen an announcement yet [1], so it looks like `1.11.33.3-beta` is still a work in progress.

[1]
https://groups.google.com/forum/#!forum/ngx-pagespeed-announce
https://developers.google.com/speed/pagespeed/module/release_notes

**Comment by crowell** [comment]
we're still getting everything finalized for the release which should be out shortly!

**Comment by nikolay** [comment]
How can we differentiate between a "released" and "unreleased" versions then? As far as I know, there's a way to properly cut out a release and mark it as a "pre-release": https://help.github.com/articles/creating-releases/

**Comment by centminmod** [comment]
released is whenever it's announced at https://groups.google.com/forum/#!forum/ngx-pagespeed-announce AFAIK

**Comment by nikolay** [comment]
@centminmod Yeah, if you're a sorry human; if you're a CI system that tests against the release a, there need to be some indication.

**Comment by crowell** [comment]
@nikolay the "releases" on github get autogenerated on tags (I haven't touched the github "releases" menu for this release for example), which isn't really ideal for us as everything can't be done atomically (updating nginx and apache modules and their distribution system requires a bit of coordination).

**Comment by jmarantz** [comment]
I'm not sure I understand the details RE git, but the request of having a
mechanical way to determine what the most recent release is without
monitoring emails is a reasonable one.

OTOH if a CI system hiccups and generates a non-working build during a
limited-time window of vulnerability in our release process, this seems
non-fatal as long as it recovers once all is updated.  Is that right?

Maybe we can do something simple like write the updated latest beta and
stable tags into a trunk files, and update those trunk files only when the
release is complete.  CI systems could read those files to figure out which
tag to use.

**Comment by nikolay** [comment]
@crowell Those aren't true releases, which are created via the API, CLI, or the web interface.
